pkg/lib/cockpit-components-table.scss

```diff
@@ -1,6 +1,4 @@
 @import "global-variables";
-@import "@patternfly/patternfly/components/Table/table.scss";
-@import "@patternfly/patternfly/components/Table/table-grid.scss";
```

Comment:
@KKoukiou Hmm, could it be that this is not actually safe yet with webpack? These were the first commits in your series, and they looked like general fixes. But there are [lots of pixel failures](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18510-20230316-051941-d8ca716c-fedora-37/log.html), and some of them are really broken, like the [top bar](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18510-20230316-051941-d8ca716c-fedora-37/pixeldiff.html#TestSuperuser-testBasic-topnav-privileged-medium)

Reply:
This can be only introduce with esbuild. I kept the commit seperate for code review ease, but eventually they must be squashes with the commit that's porting to esbuild.

Reply:
Thanks! I took it out.